pynamodb/expressions/condition.py

```diff
@@ -29,12 +29,26 @@ def __and__(self, other):
                             self.__class__.__name__, other.__class__.__name__)
         return And(self, other)
 
+    def __rand__(self, other):
+        # special case 'None & condition' to enable better syntax for chaining
+        if other is not None:
+            raise TypeError("unsupported operand type(s) for &: '{0}' and '{1}'",
+                            self.__class__.__name__, other.__class__.__name__)
```

Comment:
Did you mean to format?
```python
try:
    raise TypeError('foo: {0}', 'bar')
except Exception as ex:
    print(str(ex))
```
prints
```
('foo: {0}', 'bar')
```

Reply:
I copied the format error from above but I think it's wrong their too -- will fix all instances